tests/cli/test_cli.py

```diff
@@ -51,15 +52,29 @@ def test_helps_no_raise(self):
     def test_env_commands(self):
         subprocess.run("optimum-cli env", shell=True, check=True)
 
-    def test_export_commands(self):
```

Comment:
FYI, I have to split this because tflite is broken. Also it's not optimal to combine independent backends in a single test where failure in one would block testing other backends.

Reply:
You are totally right, we should have independent tests.

Reply:
Ok, given the required python versions are different. I decided to create new workflows for ExecuTorch with exact required setup, to avoid unintentionally bump the python version for other backends.